install_config/configuring_nsxtsdn.adoc

```diff
@@ -0,0 +1,239 @@
+nsx_insecure_ssl=true
+# vSphere Cloud Provider
+#openshift_cloudprovider_kind=vsphere
+#openshift_cloudprovider_vsphere_username='administrator@example.com'
+#openshift_cloudprovider_vsphere_password='viadmin_password'
+#openshift_cloudprovider_vsphere_host='vcsa.example.com'
+#openshift_cloudprovider_vsphere_datacenter='Example-Datacenter'
+#openshift_cloudprovider_vsphere_cluster='example-Cluster'
+#openshift_cloudprovider_vsphere_resource_pool='ocp'
+#openshift_cloudprovider_vsphere_datastore='example-Datastore-name'
+#openshift_cloudprovider_vsphere_folder='ocp'
+[masters]
+master01.example.com
+master02.example.com
+master03.example.com
+[etcd]
+master01.example.com
+master02.example.com
+master03.example.com
+[nodes]
+master01.example.com ansible_ssh_host=192.168.220.2 OpenShift_node_group_name='node-config-master' OpenShift_ip=192.168.220.2
+master02.example.com ansible_ssh_host=192.168.220.3 OpenShift_node_group_name='node-config-master' OpenShift_ip=192.168.220.3
+master03.example.com ansible_ssh_host=192.168.220.4 OpenShift_node_group_name='node-config-master' OpenShift_ip=192.168.220.4
+node01.example.com ansible_ssh_host=192.168.220.5 OpenShift_node_group_name='node-config-infra' OpenShift_ip=192.168.220.5
+#node02.example.com ansible_ssh_host=192.168.220.6 OpenShift_node_group_name='node-config-infra' OpenShift_ip=192.168.220.6
+node03.example.com ansible_ssh_host=192.168.220.7 OpenShift_node_group_name='node-config-compute' OpenShift_ip=192.168.220.7
+node04.example.com ansible_ssh_host=192.168.220.8 OpenShift_node_group_name='node-config-compute' OpenShift_ip=192.168.220.8
+----
++
+For information on the {product-title} installation parameters, see xref:../install/configuring_inventory_file.adoc#install-config-configuring-inventory-file[Configuring Your Inventory File].
+
+.Procedure
+
+. Run the following command to install required software packages, if any, for {product-title}:
++
+----
+$ ansible-playbook -i hosts openshift-ansible/playbooks/prerequisites.yml
+----
+
+. Ensure that the NCP Container image is downloaded locally on all nodes
+
+. After the playbook has successfully executed, run the following command on all nodes, replacing the `xxx` with the NCP build version:
++
+----
+$ docker load -i nsx-ncp-rhel-xxx.tar
+----
++
+For example:
++
+----
+$ docker load -i nsx-ncp-rhel-2.4.0.12511604.tar
+----
+
+. Get the image name and retag it:
++
+----
+$ docker images
+$ docker image tag registry.local/xxxxx/nsx-ncp-rhel nsx-ncp <1>
+----
+<1> Replace the `xxx` with the NCP build version. For example:
++
+----
+docker image tag registry.local/2.4.0.12511604/nsx-ncp-rhel nsx-ncp
+----
+
+. Deploy the {product-title} cluster:
++
+----
+$ ansible-playbook -i hosts openshift-ansible/playbooks/deploy_cluster.yml
+----
++
+For more information on the {product-title} installation, see xref:../install/running_install.adoc#install-running-installation-playbooks[Installing OpenShift Container Platform].
```

Comment:
s/OpenShift Container Platform/{product-title}

Reply:
@ahardin-rh Do we want to change that one to {product-title}? It is the name of the section in the link.